### PR TITLE
fixes #1705 added a 'data-index', this was used rather than the index() function

### PR DESCRIPTION
**Pull request by AaronmcL**
added a `data-index` to the handlebars template, using this rather than the `index()` function, and passing it through to `openPopup()`

### CODE REVIEW

File: js/adapt-contrib-hotgraphic.js [hunk 1]
```diff
@@ -159,16 +159,16 @@ define(function(require) {
...
-            this.openPopup();
+            this.openPopup(currentIndex);
            
             this.applyNavigationClasses(currentIndex);
         },
         
-        openPopup: function() {
-            var currentIndex = this.$('.hotgraphic-item.active').index();
+        openPopup: function(index) {
+            var currentIndex = index || this.$('.hotgraphic-item.active').index();
```
Comment (oliverfoster) [inline js/adapt-contrib-hotgraphic.js]: can the latter half of the || vanish?

Comment (sarveshwar-gavhane) [reply js/adapt-contrib-hotgraphic.js]: can first half of the || vanish and directly use index.

Comment (oliverfoster) [reply js/adapt-contrib-hotgraphic.js]: It's a good point. We don't need to redefine the index as currentIndex or use the dom.index() function at all here.

**Comment by lc-thomasberger** [comment]
I think this not needed when the itemCollection implementation is merged

**Comment by moloko** [comment]
ah - good point!

**Comment by moloko** [comment]
this has now been done as part of #147

**Pull request closed without merging**